### DISCUSSION TITLE
NVIDIA Security Bulletins

### DISCUSSION THREAD

**Discussion by madisongh** [OE4T/meta-tegra#548] [General]
Thread with information about NVIDIA's Tegra-specific security bulletins.

**Comment by madisongh** [comment]
From 25 Jan 2021: https://nvidia.custhelp.com/app/answers/detail/a_id/5147

This bulletin lists three CVEs:

CVE-2021-1070: specific to the L4T BSP rootfs creation process, does not apply when using OE/Yocto.
CVE-2021-1069: applies to the nvhost kernel driver. Patch has been back-ported from the L4T R32.5 kernel for the dunfell, dunfell-l4t-r32.4.3, gatesgarth, and master branches.
CVE-2021-1071: patches to IIO core and NVIDIA's ina3221 driver have been back-ported from the R32.5 kernel for the dunfell, dunfell-l4t-r32.4.3, gatesgarth, and master branches.

meta-tegra commits with the kernel recipe updates:
master: b0a42c95a9b9464681e72a39897a304d61aaf56d
gatesgarth: e2a61e423fb63d929da4d3725ae3f45f9ad5542b
dunfell-l4t-r32.4.3: 58a8771e833db520f5696ebf6bbcaec67bdbcb10
dunfell: 7a51e41bb8b09b7126c706c94fc822338bda8414

**Reply by ichergui** [reply]
Well done @madisongh
